packages/component/src/Composer.js

```diff
@@ -339,7 +341,7 @@ Composer.propTypes = {
       subscribe: PropTypes.func.isRequired
     }).isRequired,
     end: PropTypes.func,
-    getSessionId: PropTypes.func.isRequired,
+    getSessionId: PropTypes.func,
```

Comment:
Why was this previously required but now it is not?

Reply:
This is for OAuth. Direct Line Speech doesn't support OAuth, thus, it doesn't have `getSessionId`.

Prefer to make it optional (`undefined`) so we can do feature detection and adjust UI as needed.